### PR TITLE
Alignment with the latest version of l0-types-ext

### PR DESCRIPTION
**Pull request by italobusi**
Alignment with the latest version of l0-types-ext:
- removed identity modulation and use the identity l0-types-ext:modulation instead
- removed identity FEC and use the identity l0-types-ext:FEC

Co-authored-by: sergiobelotti <sergio.belotti@nokia.com>

### CODE REVIEW

**Comment by italobusi** [comment]
@agva123 @EstherLerouzic : this PR depends on the merge of https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext/pull/17

We have copied the ietf-layer0-types-ext@2021-02-12.yang in this repository to simplify the compilation

File: ietf-layer0-types-ext@2021-02-12.yang [hunk 1]
```diff
@@ -0,0 +1,853 @@
+module ietf-layer0-types-ext {
```
Comment (EstherLerouzic) [inline ietf-layer0-types-ext@2021-02-12.yang:1]: I think that this file should not be added here.

Comment (sergiobelotti) [reply ietf-layer0-types-ext@2021-02-12.yang:1]: that was a shortcut to permit compilation

**Comment by EstherLerouzic** [comment]
I prefer that we first merge layer0-ext to avoid adding the file here, even temporarily

**Review by EstherLerouzic — APPROVED** [review]

**Comment by sergiobelotti** [comment]
> I prefer that we first merge layer0-ext to avoid adding the file here, even temporarily

Done: , deleted layer0-type-ext from this repository and merged layer0-type-ext  https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext/pull/17

File: ietf-layer0-types-ext@2021-02-12.yang (hunk 1 above)
Comment (italobusi) [reply ietf-layer0-types-ext@2021-02-12.yang]: Fixed with https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/pull/53/commits/6be7c2d6b88b6a6ce5d5d99ab7be2396238e77ba

**Review by agva123 — APPROVED** [review]

**Pull request merged**